<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>m10 Backup</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>20</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>mtb Contents:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>50</y>
      <width>491</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>20</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Backup Folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_backupFrom">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>/home/garrett/Documents/Duplicity</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_outputFolder">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>/home/garrett/Desktop/MyOutput.mtb</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>mtb Output</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>420</y>
      <width>841</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>40</y>
      <width>21</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>120</y>
      <width>21</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>BACKUP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1088</x>
      <y>380</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract mtb Data</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>371</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Backups:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1246</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBackup">
    <property name="title">
     <string>Backup</string>
    </property>
    <addaction name="actionNew_Backup"/>
    <addaction name="actionSave_Preferences"/>
    <addaction name="actionLoad_Previous_Backup"/>
    <addaction name="actionLoad_mtb_Backup_File"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuCurrent_Backups">
    <property name="title">
     <string>Current Backups</string>
    </property>
   </widget>
   <addaction name="menuBackup"/>
   <addaction name="menuCurrent_Backups"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_Preferences">
   <property name="text">
    <string>Save Preferences</string>
   </property>
  </action>
  <action name="actionLoad_Previous_Backup">
   <property name="text">
    <string>Load Previous Backup</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoad_mtb_Backup_File">
   <property name="text">
    <string>Load mtb Backup File</string>
   </property>
  </action>
  <action name="actionNew_Backup">
   <property name="text">
    <string>New Backup</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
